hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirAclOp.java

```diff
@@ -70,14 +70,6 @@ public Object performTask() throws IOException {
         fsd.checkOwner(pc, iip);
         AclStorage.validateAclSpec(aclSpec);
         INode inode = FSDirectory.resolveLastINode(iip);
-        if (aclSpec.size() == 1 && aclSpec.get(0).getType().equals(AclEntryType.MASK)) {
-          //HOPS: We allow setting
-          FsPermission fsPermission = inode.getFsPermission();
-          inode.setPermission(new FsPermission(fsPermission.getUserAction(), aclSpec.get(0).getPermission(),
-              fsPermission
-                  .getOtherAction()));
-          return AclStorage.readINodeLogicalAcl(inode);
-        }
```

Comment:
Do you know why we were doing this?

Reply:
No idea why this code was there, let's see after testing if it affected some of the test cases. I removed it for now to be consistent with HDFS Acls